addons/website_livechat/controllers/main.py

```diff
@@ -66,3 +67,9 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, **k
         if visitor_sudo:
             anonymous_name = visitor_sudo.with_context(lang=visitor_sudo.lang_id.code).display_name
         return super(WebsiteLivechat, self).get_session(channel_id, anonymous_name, previous_operator_id=previous_operator_id, **kwargs)
+
+class LivechatWebClient(WebClient):
+    # make translations available cross-origin
+    @http.route('/web/webclient/translations/<string:unique>', type='http', auth="public", cors="*")
```

Comment:
If it's acceptable to add it in this case, it's probably acceptable to directly put it in the base module.